bedrock/pocket/templates/pocket/updates-signup.html

```diff
@@ -32,12 +32,67 @@
         }
       ),
     ) %}
-        <h1>Keep up with Pocket</h1>
-        <p>To learn about future Pocket updates and how you can be involved, please reach out to <a href="mailto:press@pocket.com">press@pocket.com</a>.</p>
-    {% endcall %}
+      <div id="pocket-updates-thanks" class="pocket-updates-success hidden">
+        <h1>{{ ftl('pocket-updates-signup-success-done-great-reads-on-way') }}</h1>
+        <p>{{ ftl('pocket-updates-signup-success-follow-on') }}</p>
+        <a class="mzp-c-button" href="/{{LANG}}/explore">
```

Comment:
Issue: this link takes me to a 404 page?

Reply:
When you're running local Pocket mode, yeah it's a 404 because getpocket.com/en/expolore is served by Pocket Core, not Bedrock. So, nothing to worry about as long as you're getting a URL that says `/en/explore`

Reply:
Would it be worth updating it to an absolute URL?

Reply:
Also: do bedrock's locales map 1:1 with the locales served by pocket? Curious if we might still get some 404's here, or if that gets handled gracefully on their end?